overlord/snapstate/autorefresh_gating_test.go

```diff
@@ -2580,13 +2580,7 @@ func (s *validationSetsSuite) TestAutoRefreshPhase1WithValidationSets(c *C) {
 	restore := snapstatetest.MockDeviceModel(DefaultModel())
 	defer restore()
 
-	refreshedDate, err := time.Parse(time.RFC3339, "2021-01-01T10:00:00Z")
-	c.Assert(err, IsNil)
-	restoreRevDate := snapstate.MockRevisionDate(func(sn *snap.Info) time.Time {
-		return refreshedDate
-	})
-	defer restoreRevDate()
-
+	refreshedDate := time.Date(2018, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Any reason why you changed this to 2018? If the test does not fail with 2021, maybe it's better to keep that, or people looking at the diff get suspicious :-)

Reply:
This is needed because of the mocking that snapmgrsSuite does in SetUpTest.

Reply:
Ah, I see! Then a minor suggestion: use the `fakeRevDateEpoch` variable, instead of repeating the date in all these places.

Reply:
Good idea, thanks.